sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/E2ETelemetryItemValidation/MetricsTests.cs

```diff
@@ -67,7 +67,7 @@ public void VerifyCounter(bool asView)
             // ASSERT
             Assert.True(telemetryItems.Any(), "Unit test failed to collect telemetry.");
             this.telemetryOutput.Write(telemetryItems);
-            var telemetryItem = telemetryItems.Single();
+            var telemetryItem = telemetryItems.LastOrDefault()!;
```

Comment:
I don't think the `OrDefault` is necessary.
In all the examples we already assert that telemetry items exist using `.Any()`